### ISSUE TITLE
no-spec-dupes fails if 2 specs have the same name in 2 different describes in same file

### ISSUE THREAD

**Issue by DamienCassou** [tlvince/eslint-plugin-jasmine#114]
```js
describe('foo1', () => {
	it('bar', () => { });
});
describe('foo2', () => {
	it('bar', () => { });
});
```

Actual: `no-spec-dupes` is triggered
Expected: `no-spec-dupes` should **not** be triggered

eslint-plugin-jasmine version: 2.2.0

**Comment by alecxe** [comment]
@DamienCassou thanks for the report.

Are you sure this is not how it should be by definition? According to the [rule docs](https://github.com/tlvince/eslint-plugin-jasmine/blob/master/docs/rules/no-spec-dupes.md#disallow-the-use-of-duplicate-spec-names-no-spec-dupes):

```js
describe("Unique parent", function(){
  it("Same spec name", function() {});
});
describe("Different parent", function(){
  it("Same spec name", function() {});
});
```

**Comment by DamienCassou** [comment]
Thanks for the link. Apparently, I'm looking after what they call the "branch mode".

**Comment by DianaSuvorova** [comment]
Sounds like this is a no-issue. Doesn't look like any action is required. Should we close it, @burabure ?
